src/System.ObjectModel/src/System/Collections/ObjectModel/KeyedCollection.cs

```diff
@@ -126,7 +127,8 @@ private bool ContainsItem(TItem item)
                 return Items.Contains(item);
             }
 
-            if (dict.TryGetValue(key, out TItem itemInDict))
+            TItem itemInDict;
```

Comment:
If the declaration is inline you can't bang it away?

Reply:
Yes, I was not able to bang, even with brackets

Reply:
Interesting.